### DISCUSSION TITLE
Repair Items?

### DISCUSSION THREAD

**Discussion by KalCorp** [ldtteam/minecolonies#9396] [Ideas]
Anyone look into repair items? 
Example couple of Bows in storage that are not 100%,  Can be combined to make a new one at 100%?

Its easy to do in crafting table  but you cant assign that to a fletcher..

**Comment by Thodor12** [comment]
This is too difficult for the request system to manage.

The crafting recipe would just be "combine 2 bows together", well, when do you do that?
If it has 1 tick of damage it could already considered be repairable, which is not a good idea obviously.

Eventually those bows will get used up, or just repair them yourself.
